### PR TITLE
Bump version to 1.2.0-rc2

### PR DESCRIPTION
**Pull request by dongluochen**
Signed-off-by: Dong Chen <dongluo.chen@docker.com>

### CODE REVIEW

**Comment by dongluochen** [comment]
ping @vieux @nishanttotla.

**Comment by dongluochen** [comment]
Test failure unrelated. Let's fix them after.

**Comment by vieux** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 - Differentiate constraint errors from affinity errors
 - Printing unsatisfiable constraints for container scheduling failure
+- Enable rescheduling on master manager to prevent replica managers from rescheduling containers
+- Output error when starting a rescheduled container fails
 
 #### API
 
 - Introduce engine-api client to Swarm
 - Implement 'info' and 'version' with engine-api
+- Use apiClient for some volumes, network, image operations
```
Comment (nishanttotla) [inline CHANGELOG.md]: nit: `volumes` -> `volume` just to be consistent.

**Comment by nishanttotla** [comment]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (dongluochen) [inline CHANGELOG.md]: Updated.

**Pull request merged**